docs/resources/user.md

```diff
@@ -27,15 +27,18 @@ resource "wallarm_user" "user" {
 
 ## Argument Reference
 
-* `email` - (Required) User email. The value will be used as the username for authentication in Wallarm Console.
-* `realname` - (Required) The first and last name of the user.
-* `permissions` - (Required) User role. Can be one of: `admin`, `analyst`, `deploy`, `read_only`, `global_admin`, `global_analyst`, `global_read_only`. Roles description is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/settings/users/#user-roles).
-* `password` - (Optional) User password. If the value is not specified, it will be generated automatically and returned in the attribute `generated_password`.
-* `phone` - (Optional) User phone number.
-* `client_id` - (Optional) ID of the client to apply the rules to. The value is required for multi-tenant scenarios.
+* `email` - (**required**) user email. The value will be used as the username for authentication in Wallarm Console.
+* `realname` - (**required**) the first and last name of the user.
+* `permissions` - (**required**) user role. Can be one of: `admin`, `analyst`, `deploy`, `read_only`, `global_admin`, `global_analyst`, `global_read_only`. Roles description is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/settings/users/#user-roles).
+* `password` - (optional) user password. If the value is not specified, it will be generated automatically and returned in the attribute `generated_password`.
+* `phone` - (optional) user phone number.
```

Comment:
is this field still actual?

Reply:
I believe, it's still possible to provide a phone number when creating a user through the API. However, it probably does nothing in the Cloud.